### PR TITLE
KOGITO-9617: Remove the Extended Services from Serverless Logic Web Tools

### PR DESCRIPTION
**Pull request by fantonangeli**
Jira: https://issues.redhat.com/browse/KOGITO-9617

❗ Note: This PR depends on https://github.com/kiegroup/kie-tools/pull/1872, please merge after this. :exclamation: 

**Description:**
The only reason we need the Extended Services in the Serverless Logic Web Tools is to use the CORS Proxy we have inside it to interact with OpenShift and Service Registries without being blocked by CORS policies.

After [this PR](https://github.com/kiegroup/kie-tools/pull/1824) [,](https://github.com/kiegroup/kie-tools/pull/1824,) the CORS Proxy will be moved from the Extended Services to a new package, which will be deployed in favor of our current [git-cors-proxy](https://git-cors-proxy-kie-sandbox.rhba-0ad6762cc85bcef5745bb684498c2436-0000.us-south.containers.appdomain.cloud/) deployment.

The goal of this task is to remove the Extended Services from Serverless Logic Web Tools, which includes making changes in the code itself, quickstart guides, env vars, and GitHub workflows.

Since we have many references of the Extended Services in the [docs](https://github.com/kiegroup/kogito-docs), updating them is also part of this task.

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-openshift-integration.adoc [hunk 1]
```diff
@@ -71,9 +45,4 @@ If the entered values are correct, then the tab updates and displays *You're con
 
 After connecting to OpenShift, you are ready to deploy your {product_name} projects using {serverless_logic_web_tools_name}. For more information about deploying your projects, see xref:tooling/serverless-logic-web-tools/serverless-logic-web-tools-deploy-projects.adoc[Deploying your {product_name} projects using {serverless_logic_web_tools_name}].
 
-[NOTE]
-====
-If your OpenShift instance uses self-signed certificates, then you must enable the `InsecureSkipVerify` on Extended Services.
```
Comment (caponetto) [inline serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-openshift-integration.adoc:76]: Please add this information back just like we have on the Quickstart.

**Review by caponetto — COMMENTED** [review]
Thanks @fantonangeli!
Can you please update the images that contain the Extended Services icon?

**Review by ricardozanini — APPROVED** [review]

**Review by caponetto — APPROVED** [review]

**Comment by caponetto** [comment]
Please hold the merge of this PR until we are closer to kie-tools `0.32.0` release.

**Review by MarianMacik — APPROVED** [review]

**Comment by caponetto** [comment]
As KIE Tools `0.32.0` release is planned for this week, this PR is ready to be merged. Thanks!

**Comment by ricardozanini** [comment]
@krisv @ederign please approve

**Review by ederign — APPROVED** [review]

**Comment by caponetto** [comment]
@kaldesai We've only removed unnecessary information here. Could you please take a look?

File: serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-openshift-integration.adoc [hunk 2]
```diff
@@ -65,15 +39,11 @@ image::tooling/serverless-logic-web-tools/serverless-logic-web-tools-openshift-i
 . Enter your OpenShift project name in the *Namespace (project)* field.
 . Enter the value copied value of `--server` flag in the *Host* field.
 . Enter the value of API token in the *Token* field.
+. If your OpenShift instance uses Self-Signed certificates, then you must also check the Insecurely disable TLS certificate validation option.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-openshift-integration.adoc:42]: Is `Insecurely disable TLS certificate validation` a UI element? If yes, then please make it bold. 

```suggestion
. If your OpenShift instance uses Self-Signed certificates, you must also check the *Insecurely disable TLS certificate validation* option.
```

**Review by kaldesai — APPROVED** [review]
All the changes look good to me. Just one minor suggestion. Thank you!

**Comment by fantonangeli** [comment]
> All the changes look good to me. Just one minor suggestion. Thank you!

Thank you @kaldesai for your suggestion

**Pull request merged**